src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractReportServiceImpl.java

```diff
@@ -0,0 +1,325 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.io.File;
+import java.util.List;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.report.ConcurBatchReportLineValidationErrorItem;
+import edu.cornell.kfs.concur.batch.report.ConcurBatchReportSummaryItem;
+import edu.cornell.kfs.concur.batch.report.ConcurRequestExtractBatchReportData;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractReportService;
+import edu.cornell.kfs.sys.service.ReportWriterService;
+
+public class ConcurRequestExtractReportServiceImpl implements ConcurRequestExtractReportService {
```

Comment:
There are a couple of long functions in this class.  Can they be refactored per clean code.